test/CMakeLists.txt

```diff
@@ -35,13 +35,12 @@ project_options(
   ENABLE_CPPCHECK
   ENABLE_CLANG_TIDY
   # ENABLE_INCLUDE_WHAT_YOU_USE
-  ENABLE_COVERAGE
-  ENABLE_PCH
-  PCH_HEADERS
-  ${PCH_HEADERS}
+  # TODO(CK: should not installed with this option!) ENABLE_COVERAGE
+  # TBD: ENABLE_PCH
+  # TBD: PCH_HEADERS ${PCH_HEADERS}
   ENABLE_DOXYGEN
-  ENABLE_INTERPROCEDURAL_OPTIMIZATION
-  ENABLE_NATIVE_OPTIMIZATION
+  # ENABLE_INTERPROCEDURAL_OPTIMIZATION
+  # ENABLE_NATIVE_OPTIMIZATION
```

Comment:
These should be all reverted,

Reply:
There were build errors on OSX, no idea why!
**The linker crashed while build release.**